### ISSUE TITLE
FeatureImp with AUC loss?

### ISSUE THREAD

**Issue by arbet003** [christophM/iml#154]
Hello,

I noticed your book has an example using "1-AUC" loss for permutation feature importance in a binary classification problem (section 5.5.3 of your [online book](https://christophm.github.io/interpretable-ml-book/feature-importance.html)).  

However, I am having trouble figuring out how to use AUC (or 1 - AUC) with the FeatureImp() function.  The Metrics R package does have "auc" as an option, but I get an error when trying to loss = "auc" with FeatureImp.  Is auc currently not supported?

Thanks

**Comment by corneliagru** [comment]
Hi,
I'm having the same problem. Are there any news on this?
Thanks!

**Comment by christophM** [comment]
The solution is to provide a function to FeatureImp.

Here you found how I did the AUC example: https://github.com/christophM/interpretable-ml-book/blob/master/manuscript/05.6-agnostic-permfeatimp.Rmd
